src/js/common/components/RequiredLoginView.jsx

```diff
@@ -36,53 +55,50 @@ class RequiredLoginView extends React.Component {
   }
 
   renderVerifiedContent = () => {
-    const { serviceRequired, user } = this.props;
+    if (this.shouldVerify()) {
+      return <LoadingIndicator setFocus message="Redirecting to verify..."/>;
```

Comment:
This and the "Redirecting to login..." loaders should never actually be shown since the user should be redirected before then. But a placeholder nonetheless if it's somehow not happening right away, and easy to catch if it hangs in tests.